self-api/src/main/java/com/selfxdsd/api/Invoice.java

```diff
@@ -13,6 +13,9 @@
  * @author criske
  * @version $Id$
  * @since 0.0.3
+ *
+ * @todo #979:30min Add payments() method that returns Invoice's Payments and
+ * its implementation in StoredInvoice.
```

Comment:
@fellahi-ali second and rest of sentences should always start with a space :D

Reply:
ah, sorry, I know that but don't know how I missed it :(